package.json

```diff
@@ -13,14 +13,14 @@
     "babel-plugin-transform-flow-strip-types": "^6.22.0",
     "babel-plugin-webpack-loaders": "0.9.0",
     "babel-polyfill": "^6.23.0",
-    "babel-preset-es2015-loose": "^8.0.0",
-    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-env": "^1.6.1",
```

Comment:
@alirezavalizade Thank you for submitting this PR!

Updating `package.json` should have resulted in an updated `yarn.lock` file.

Did you run the following?
```
yarn remove babel-preset-es2015-loose babel-preset-es2015
yarn add --dev babel-preset-env
```

Reply:
No i'm not modified yarn.lock, I will fix that sooner.